src/components/course/index.jsx

```diff
@@ -1,12 +1,151 @@
 import React from 'react';
+import styled from '@emotion/styled';
 import { useParams } from 'react-router-dom';
-import { Text } from '@chakra-ui/react';
+import courseData from '../../constants/courseData';
 
 function Course() {
   const { courseId } = useParams();
+  const data = courseData[courseId];
+  function importAll(r) {
+    return r.keys().map(r);
+  }
+  const images = importAll(
+    require.context('../../static/pic', false, /\.(png|jpe?g|svg)$/),
+  );
+
   return (
-    <Text>Course id: {courseId}</Text>
+    <CourseContainer>
+      <InfoContainer>
+        <CourseName>{data.courseName}</CourseName>
+        <Description>{data.courseDesc}</Description>
+      </InfoContainer>
+      {data.sourceList.map((course) => (
+        <CourseCardContainer key={course.title}>
+          <Information>
+            <Row style={{ color: '#000000' }}>
+              <Circle
+                style={{
+                  backgroundColor: `${
+                    course.category === '影片' ? '#64379F' : '#37999F'
+                  }`,
+                }}
+              />
+              {course.category}
+            </Row>
+            <Row style={{ fontSize: '0.94rem' }}>{course.platform}</Row>
+            <Row style={{ fontSize: '1.25rem' }}>{course.title}</Row>
+            <Row>
+              <b>課程來源：</b>
+              <CourseLink href={course.link}>{course.link}</CourseLink>
+            </Row>
+            <Row>
+              <b>演講者：</b>
+              {course.presenter}
+            </Row>
+            <Row>{course.description}</Row>
+          </Information>
+          <Source
+            src={
+              images.find((e) =>
+                e.default.includes(course.pictureName.split('.')[0])
+                  .default
```

Comment:
Missing a `)` here.

Reply:
Done in a9fb157065ef82c6b9c60c6702a620bb0bf1389b